docs/moduleCom1DFA.rst

```diff
@@ -43,21 +43,29 @@ In the directory ``Inputs``, the following files are required:
 * release area scenario as shapefile (in Inputs/REL); multiple features are possible
   -> the release area name should not contain an underscore, if so '_AF' is added
 
-.. Note:: Release thickness can be specified in the shape file by adding an attribute called `thickness` or via the
-          configuration file (see :ref:`moduleCom1DFA:Model configuration`.) -> in this case you need to set the flag
-          `useRelThFromIni` to True! For backwards compatibility, the attribute 'd0' also works, but we suggest
-          to use `thickness` in new projects.
-
-
 and the following files are optional:
 
 * entrainment area as shapefile (in Inputs/ENT)
 * resistance area as shapefile (in Inputs/RES)
 * secondary release area as shapefile (in Inputs/SECREL)
 
-.. Note:: Entrainment thickness can be specified in the shape file by adding an attribute called `thickness` or via the
-          configuration file (see :ref:`moduleCom1DFA:Model configuration`.) -> in this case you need to set the flag
-          `useEntThFromIni` to True!
+.. Note:: Release, entrainment and secondary release thickness can be specified in two different ways.
+          For one, thickness can be specified in the shape file by adding an attribute called
+          `thickness` -> in this case you need to set the flag `thicknessFromShp` to True in the
+          configuration file (which is the default setting). For backwards compatibility,
+          the attribute 'd0' also works, but we suggest to use `thickness` in new projects. If this
+          setting is chosen - ALL features in a shape file have to provide a thickness value.
+          Parameter variation can be added in the `PercentVariation` parameter in the configuration file,
+          using ``+-percentage$numberOfSteps``. If you provide `+` a positive variation will be performed,
+          if `-` is given, only a negative variation is performed, whereas if no sign is prefixed, variation
+          will be performed in both directions in the given number of steps.
+          Secondly, the thickness can be read directly from the configuration file -> in this case you
+          need to set the flag 'thicknessFromShp' to False and provide your desired thickness value
+          in the respective thickness parameter (relTh, entTh or secondaryRelth). Regarding parameter
+          variations, in addition to the `PercentVariation` option, the standard variation options in
+          :ref:`configuration:Configuration`, you can also directly set e.g. `relTh = 1.$50$2`,
```

Comment:
does the option with the 0.5:1.5:2   or 0.5|1|1.5 still work?

Reply:
yes if relThFromShp=False, you can use those for relTh=..